src/frontend/components/Felleskomponenter/LeggTilKnapp/LeggTilKnapp.tsx

```diff
@@ -17,29 +17,25 @@ interface Props {
     id?: string;
 }
 
-const StyledFlatKnapp = styled(Flatknapp)<{ feilmelding }>`
-    margin: 0.5rem 0 0.5rem 0;
+const StyledButton = styled(Button)<{ className }>`
     && {
-        white-space: normal;
-        text-align: left;
-        border: ${props => (props.feilmelding ? `2px solid ${navFarger.redError}` : 'none')};
+        margin: 0.5rem 0 0.5rem 0;
+        border: ${props => (props.className ? `2px solid ${navFarger.redError}` : 'none')};
     }
 `;
 
 export const LeggTilKnapp: React.FC<Props> = ({ onClick, språkTekst, feilmelding, id }) => (
     <>
-        <StyledFlatKnapp
+        <StyledButton
             id={id}
-            feilmelding={feilmelding ? feilmelding : undefined}
-            htmlType={'button'}
-            kompakt
+            variant={'tertiary'}
+            type={'button'}
             onClick={onClick}
+            className={feilmelding ? feilmelding.toString() : ''}
```

Comment:
Trenger vi å sette et klassenavn for å få dette til?🤔

Reply:
Har lagt inn et forslag nå